docs/rackhd/skus.rst

```diff
@@ -215,3 +260,42 @@ SKUs are defined via JSON, with these required fields:
 +------------------------+-----------+--------------------------+----------------------------------------------------------+
 | discoveryGraphOptions  |  Object   | *optional*               | Options to pass to the graph being run on node discovery.|
 +------------------------+-----------+--------------------------+----------------------------------------------------------+
+
+SKU Pack tar.gz format
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The SKU pack requires the 'config.json' to be at the root of the tar.gz file.  A typical
+package may have a static and template directory.
+
+.. code-block:: shell
+
+    tar tzf pack.tar.gz:
+    config.json
+    static/
+    static/common/
+    static/common/discovery.overlay.cpio.gz

```

Comment:
This just defaults to http (`/static/http/common`)? So tftp file overrides aren't supported currently?